### DISCUSSION TITLE
Edit the single Product WooCommerce page

### DISCUSSION THREAD

**Discussion by bnikkhah** [holger1411/understrap#1609] [Q&A]
Hey guys!

I've used Understrap for years and I'm proud to say that I'm still using it today.

I'm working on a new project with someone and I'm wondering which file I should go to in order to edit a single product WooCommerce template. Technically, I can go to my _Understrap Child_ theme and make a file named `single-product.php`, but then I would start out with an empty template without the pre-built Understrap template that comes bundled with the theme. I'm simply searching for a way to copy and paste the pre-existing single product template, put it inside my child theme, and then make additional edits from there.

The only problem I'm facing is that I simply cannot find where the file is located in the Understrap theme! 😅

May someone guide me to the right direction? Thank you for your help!

**Comment by bacoords** [comment]
I recommend using the Query Monitor plugin - which will show you every template php file that's being loaded anywhere on your site. 

But yes, for the overall product page, you can duplicate the single.php file from the parent theme, rename it single-product.php and start modifying from that.
